packages/notifications/src/elements/toaster/ToastProvider.tsx

```diff
@@ -30,9 +37,19 @@ export interface IToastProviderProps {
    * Sets the `z-index` of the toast
    */
   zIndex?: number;
+  /**
+   * Props to be spread onto the wrapping element for each Toast placement.
+   * Can be used to customize the positioning of a placement.
```

Comment:
Rewrite to follow documentation guidelines (present simple, third person singular, no period) and consistent with existing prop descriptions seen on the `website`:

"Passes placement-based customization props to the toast's parent element"

Reply:
Updated.